KWZP2019/KWZP2019/ProductionProce.cs

```diff
@@ -0,0 +1,33 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated from a template.
+//
+//     Manual changes to this file may cause unexpected behavior in your application.
+//     Manual changes to this file will be overwritten if the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace KWZP2019
+{
+    using System;
+    using System.Collections.Generic;
+    
+    public partial class ProductionProce
```

Comment:
Powinno być `ProductionProcess`, nieprawdaż?

Reply:
To nie moja klasa.

Reply:
dodana w tym PR, proszę poprawić